src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -104,7 +91,9 @@
     /**
      * The MineColonies achievement page
      */
-    public static final AchievementPage pageMineColonies = new AchievementPageMineColonies(achGetSupply, achWandOfbuilding, achBuildingTownhall, achBuildingBuilder, achBuildingColonist, achBuildingLumberjack, achBuildingMiner, achBuildingFisher, achSizeSettlement, achSizeTown, achsizeCity, achUpgradeColonistMax, achUpgradeBuilderMax, achUpgradeLumberjackMax, achUpgradeMinerMax, achUpgradeFisherMax);
```

Comment:
Can you, for clean looks, format it like this:
```
public static final AchievementPage pageMineColonies = new AchievementPageMineColonies(
    achGetSupply, achWandOfbuilding, .... //lets say 5 or 6 per line so that it looks good
);
```